tests/testthat/test-ci.R

```diff
@@ -51,3 +51,12 @@ patrick::with_parameters_test_that(
   },
   env_var_value = list("", "F", NA, NULL)
 )
+
+test_that("GitHub Actions log is skipped in pkgdown websites", {
+  withr::local_envvar(list(GITHUB_ACTIONS = "true", IN_PKGDOWN = "true"))
+  withr::local_options(lintr.rstudio_source_markers = FALSE)
+  tmp <- withr::local_tempfile(lines = "x <- 1:nrow(y)")
+
+  l <- lint(tmp)
```

Comment:
Might it be better to set `linters=seq_linter()` explicitly? Feels more future-proof & intentional.